strr-web/tests/unit/components/stepper/Stepper.spec.ts

```diff
@@ -48,3 +48,95 @@ it('can mount Stepper Footer component', async () => {
     })
   expect(stepper.find('[data-test-id="stepper-footer"]').exists()).toBe(true)
 })
+
+describe('BcrosStepper', () => {
+  const mockSteps = [
+    {
+      step: {
+        label: 'Step 1',
+        inactiveIconPath: '/inactive1.svg',
+        activeIconPath: '/active1.svg',
+        complete: false,
+        isValid: false,
+        alt: 'Step 1'
+      },
+      title: 'Title 1',
+      subtitle: 'Subtitle 1',
+      formTitle: 'Form 1',
+      sections: []
+    },
+    {
+      step: {
+        label: 'Step 2',
+        inactiveIconPath: '/inactive2.svg',
+        activeIconPath: '/active2.svg',
+        complete: true,
+        isValid: true,
+        alt: 'Step 2'
+      },
+      title: 'Title 2',
+      subtitle: 'Subtitle 2',
+      formTitle: 'Form 2',
+      sections: []
+    }
+  ]
+
+  it('displays correct number of steps', async () => {
+    const wrapper = await mountSuspended(BcrosStepper, {
+      global: { plugins: [i18n] },
+      props: {
+        steps: mockSteps,
+        activeStep: 0
+      }
+    })
+
+    const stepElements = wrapper.findAll('[aria-roledescription="button"]')
+    expect(stepElements.length).toBe(mockSteps.length)
+  })
+
+  it('displays active step correctly', async () => {
+    const wrapper = await mountSuspended(BcrosStepper, {
+      global: { plugins: [i18n] },
+      props: {
+        steps: mockSteps,
+        activeStep: 1
+      }
+    })
+
+    const activeStepElement = wrapper.findAll('[aria-roledescription="button"]')[1]
+    expect(activeStepElement.classes()).toContain('border-b-[3px]')
```

Comment:
What is the purpose of this test?
I would not test against the css classes that could be easily changed and run into failed cases. If you want to test if step is active you can create a dynamic data test id, eg. `:data-test-id="isActiveStep ? 'step-active' : 'step-inactive'"` so it could be tested without affecting styles or anything else.

Reply:
Refactored to use data-test-ids.